src/main/java/uk/gov/hmcts/reform/bulkscanning/model/enums/EnvelopeSource.java

```diff
@@ -2,6 +2,6 @@
 
 public enum EnvelopeSource {
     Bulk_Scan,
-    Exela,
+    Fee_and_Pay,
```

Comment:
Hi @subrotomukherjee ,

As discussed, It will have a wider impact for all the Exela data that is coming through please make sure only label is changed in the **data_loss** report.